hack/update/cilium_version/cilium_version.go

```diff
@@ -45,14 +45,20 @@ func main() {
 	version := strings.TrimPrefix(stable.Tag, "v")
 
 	// Add the cilium repo to helm
-	if err := exec.Command("helm", "repo", "add", "cilium", "https://helm.cilium.io/").Run(); err != nil {
-		klog.Fatal(err)
+	addRepoCmd := exec.Command("helm", "repo", "add", "cilium", "https://helm.cilium.io/")
+	if err := addRepoCmd.Run(); err != nil {
+		klog.Fatalf("failed to run command %v: %v", addRepoCmd.Args, err)
+	}
```

Comment:
Why is this better? We could use Fatalf() without adding a variable for the command.

Since we run lot of commands in the same way, it will help to use a common Run() function so we can have:

```go
if err := Run("helm", "repo", "add", "cilium", "https://helm.cilium.io/"); err != nil {
    handle it ...
}
```

This ensures that we run everything with the same way (stdout, stderr, env). It also can return better errors - the error we get from command.Run() is usually not helpful (command exited with error code 1). We typically want to get the ExitError.Stderr to report meaningful errors.